namespaces/live-1.cloud-platform.service.justice.gov.uk/poornima-dev/resources/rds.tf

```diff
@@ -0,0 +1,58 @@
+/*
+ * When using this module through the cloud-platform-environments, the following
+ * two variables are automatically supplied by the pipeline.
+ *
+ */
+
+variable "cluster_name" {}
+
+variable "cluster_state_bucket" {}
+
+/*
+ * Make sure that you use the latest version of the module by changing the
+ * `ref=` value in the `source` attribute to the latest version listed on the
+ * releases page of this repository.
+ *
+ */
+module "cp_team_test_rds" {
+  source                 = "github.com/ministryofjustice/cloud-platform-terraform-rds-instance?ref=4.3"
+  cluster_name           = "${var.cluster_name}"
+  cluster_state_bucket   = "${var.cluster_state_bucket}"
+  team_name              = "cp-team-test-repo"
+  business-unit          = "cp-team-test-bu"
+  application            = "cpteamtestapp"
+  is-production          = "false"
+  environment-name       = "development"
+  infrastructure-support = "poornima.krishnasamy@digtal.justice.gov.uk"
+  force_ssl              = "true"
```

Comment:
You might have to do a bit of extra work to get this working with SSL. I think you'll need to download and install a certificate authority file from Amazon. Raz has the details.
If this is just to play around with, it may be easier to turn force_ssl off initially, and add it in once you're confident everything else is working correctly. But, YMMV.

Reply:
I have turned the force_ssl off as of now. But will definitely give a go later